BUILD.md

```diff
@@ -147,6 +147,7 @@ git sparse-checkout set recipes/snappy
 git sparse-checkout add recipes/soci
 git fetch origin master
 git checkout master
+conan export --version 2.4.1 external/wamr # TODO: needs to be added to the conan center index
```

Comment:
We've just removed all conan packaging code from external, let's not add it back please.
Could you please create this PR in http://github.com/XRPLF/conan-center-index?

It's our fork of official CCI, so we will be reviewing code, not Conan team

Reply:
https://github.com/XRPLF/conan-center-index/pull/17, please add it to conan